README.md

```diff
@@ -527,7 +527,30 @@ The database name for the create script gets resolved in the following manner:
 1. Package name.
 > Note: 
 >- the generated script also uses the resolved database name via a setvar command.
->- if `IncludeCompositeObjects` is true, the composite objects (tables, etc.) from external references are also included in the generated script. This property defaults to `true`   
+>- if `IncludeCompositeObjects` is true, the composite objects (tables, etc.) from external references are also included in the generated script. This property defaults to `true`
+
+## Static code analysis
+Starting with version 2.7.0 of the SDK, there is support for running static code analysis during build. The SDK includes the following sets of rules:
```

Comment:
Can users can also add their own rules by placing them in the correct folder? If so, we should mention that somewhere. If not, you can ignore this! :slightly_smiling_face:

Reply:
Yeah, it is therotically possible:
- build a strong named rules .dll that targets net6.0 (normally it would target net462)
- copy to the tools folder in the NuGet cache (where Dacpactool.dll is located) - might be complex/not possible on a build server?

But it is not really something I want to support.

I described an alternative approach in the [issue comments](https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/issues/170#issuecomment-1962882435), but I want your feedback and user demand first